core/src/main/scala/cats/syntax/option.scala

```diff
@@ -27,6 +27,17 @@ final class OptionIdOps[A](val a: A) extends AnyVal {
 }
 
 final class OptionOps[A](val oa: Option[A]) extends AnyVal {
+  /**
+    * If the `Option` is `Some` return its value in a [[scala.util.Right]].
+    * If the `Option` is `None`, return the provided `B` value in a [[scala.util.Left]].
```

Comment:
> r] /home/travis/build/typelevel/cats/core/src/main/scala/cats/syntax/option.scala:30: Could not find any member to link for "scala.util.Right".
[error]   /**
[error]   ^
[error] one error found
[error] (docs/scalaunidoc:doc) Scaladoc generation failed


linking to external types fails, I would just use `scala.util.Right/Left` instead.